components/speedreader/speedreader_switches.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_SPEEDREADER_SPEEDREADER_SWITCHES_H_
+#define BRAVE_COMPONENTS_SPEEDREADER_SPEEDREADER_SWITCHES_H_
+
+namespace speedreader {
+
+constexpr char kEnableSpeedreader[] = "enable-speedreader";
```

Comment:
these should be defined in a cc file

Reply:
why?

Reply:
I guess there is a mix of `constexpr char` and `extern const` in the chromium code and the style guide doesn't even mention `extern const` anywhere, only `constexpr` so I guess that's preferred https://google.github.io/styleguide/cppguide.html#Use_of_constexpr